tcms/testcases/templates/testcases/mutable.html

```diff
@@ -100,6 +100,63 @@
                 </div>
             </div>
 
+            <div class="form-group">
+                <div>
+                    <label class="col-md-1 col-lg-1" >{% trans "Setup Duration" %}</label>
+                        <div class="col-md-3 col-lg-3">
+                            <div class="col-md-3 col-lg-3">
+                                 <input type="text" class="form-control" id="duration4" value="86400">
+                                 <label class="duration-label4">Days</label>
```

Comment:
The current HTML widgets don't account for the fact that the new fields are also form fields and in case of errors they may have a previously selected value. See how other fields are referenced in the HTML file.